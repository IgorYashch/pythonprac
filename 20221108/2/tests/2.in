aj;ldfjad
(True, False), (False, False), (True, True)
aldsfjlkdsajfl
1, 2, 3, 4, (5, 6)
(0.5, 1.3), (12.4, 7.6), (20, 0)
sdajfldsasafd
