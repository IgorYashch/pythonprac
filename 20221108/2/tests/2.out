Invalid input
Not a triangle
Invalid input
Invalid input
69.16
